packages/forms/src/UIForm/fields/File/File.component.js

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate } from 'react-i18next';
+import FieldTemplate from '../FieldTemplate';
+import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
+
+import { I18N_DOMAIN_FORMS } from '../../../constants';
+import theme from './File.scss';
+
+/**
+ * Extract the file name from the data url
+ * @param {String} value The base64 value of the file.
+ * Looks like `data:text/xml;name=test.xml;base64,PD94bWwgdmVyc2l...`
+ * @return {String} The file name, for exemple: `test.xml`.
+ */
+function getFileName(value) {
+	if (value && value.indexOf(';name=') !== -1) {
+		return value.slice(value.indexOf(';name=') + 6, value.indexOf(';base64,'));
```

Comment:
`6` should be extracted as `";name="` length.

Reply:
`";name="` and `";base64="` are reused on multiple places. It might be a good idea to extract them as constants.